### ISSUE TITLE
Please mention screenreader visibility utilities

### ISSUE THREAD

**Issue by royvanv** [tailwindcss/docs#311]
The [screenreader visibility utilities](https://github.com/tailwindcss/tailwindcss/releases/tag/v1.1.0#added-utilities-for-screenreader-visibility) seem to be missing from the docs.

**Comment by adamwathan** [comment]
Yeah been meaning to get to this, a PR to get it started would be welcome for sure.

**Comment by royvanv** [comment]
I've added PR #312.
